building-web-forms/form-and-form-elements/README.md

```diff
@@ -0,0 +1,67 @@
+# Script
+
+In this video, we will learn about HTML Forms and HTML elements used within it to collect user input in web pages.
```

Comment:
we will learn about HTML Form and HTML elements used within it, to collect user input in web pages